ui/.eslintrc.json

```diff
@@ -9,5 +9,7 @@
     },
     "env": {
         "browser": true
-    }
+    },
+    "plugins": ["plugin:patternfly-react/recommended"]
```

Comment:
This should not be in `plugins`, but in `extends` (bug in patternfly's docs). Also the order of `extends` items should be like this:
```
"extends": [
        "airbnb",
        "plugin:patternfly-react/recommended"
    ],
```

Reply:
+1